arbitrum-docs/for-devs/quickstart-solidity-hardhat.md

```diff
@@ -365,7 +370,9 @@ Select `Arbitrum Goerli` from Metamask's dropdown, paste your contract address i
 
 ### Deploy the smart contract to Arbitrum One Mainnet
 
+Now that we've verified that our smart contract works on Arbitrum's Goerli testnet, we're ready to deploy it to Arbitrum One Mainnet. This is the same process as deploying to Arbitrum's Goerli testnet, except that we'll need to pay a transaction fee in real $ETH instead of $AGOR.
```

Comment:
We could add `(Note: You may see a wildly inconsistent gas usage, this is because our 2-d gas model, for more details, you can see [here](../faqs/gas-faqs.mdx))`